.travis.yml

```diff
@@ -15,7 +15,7 @@ env:
 script:
   - docker info
   - docker run --rm --privileged multiarch/qemu-user-static:register --reset
-  - docker build --build-arg VCS_REF=$TRAVIS_COMMIT --build-arg BUILD_DATE=$(date +"%Y-%m-%dT%H:%M:%SZ") -t $DOCKER_REPO:$TRAVIS_BRANCH-$TARGET -f extra/Dockerfile${EXT} .
+  - travis_wait 40 docker build --build-arg VCS_REF=$TRAVIS_COMMIT --build-arg BUILD_DATE=$(date +"%Y-%m-%dT%H:%M:%SZ") -t $DOCKER_REPO:$TRAVIS_BRANCH-$TARGET -f extra/Dockerfile${EXT} .
```

Comment:
Why is `travis_wait 40` needed?

Reply:
I had to add this since compiling lxml for arm takes longer than 20 minutes. Travis times out after 10. Qemu emulation is making this so slow i guess.